modules/entity_to_text_tika/src/Storage/PlaintextStorage.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\entity_to_text_tika\Storage;
+
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\StreamWrapper\StreamWrapperManager;
+use Drupal\Core\StreamWrapper\StreamWrapperManagerInterface;
+use Drupal\file\Entity\File;
+
+/**
+ * Provide Capabilities to store a Text content to plain-text file.
+ */
+class PlaintextStorage {
+
+  public const DESTINATION = 'private://entity-to-text/ocr';
```

Comment:
I'm fine with the constant right now if in the future we could convert this into a Setting. Do you think it will be possible without breaking the API?

Reply:
> I'm fine with the constant right now if in the future we could convert this into a Setting. Do you think it will be possible without breaking the API?

I will already move it to a setting for local

Reply:
Well thinking about it, I will not move it into a settings.

If moved, then we should create a chain of potential storage (plain, remote, ...) and allow user to enable/disable the ones it wants and therefore allow settings.
Plusing including by default the caching capabilities inside the module instead of having it to be implemented by developer.

Therefore, I would prefer keep it as simple as possible for now and if used by community, having another approch with more capabilities